package/com.unity.formats.usd/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 ### Bug Fixes
 - Fixed an import bug causing instanced primitives not to be sanitized.
 - Disabled plugins on unsupported platforms.
+- Fixed a bug causing PointInstances to be duplicated.
```

Comment:
Might be worth also adding the time fix to the changelog

Reply:
Good point, thanks!

Reply:
Can we include some of the text of the exception itself so a user can find that in the CHANGELOG?

Reply:
Done :)